### DISCUSSION TITLE
Roborock S7 map deleted

### DISCUSSION THREAD

**Discussion by smesscle** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#727] [General]
Hi to all,
Somehow, by using this wonderful lovelace-xiaomi-vacuum-map-card, my vacuum maps have been deleted.
The card is showing fail to retrieve the map.
In the Xiaomi mobile app, the map are there but are without any cartography elements (no wall, no anything).

Even when I ask the robot to re-learn the environment, nothing is being displayed/shown on the Xiaomi mobile app.

seems I will have to do a hard reset of the robot.

Any one faced something like this before?

Thanks,

**Comment by smesscle** [comment]
To emphasise, I have re-learn my appartment with the Roborock mobile app and it seems working (I can see the map). When I connect the vacuum to the Xiaomi app, the map is blank (and the vacuum feel lonely without wall to bump into).

Any idea how to fixed this?

Thanks for the support.

**Comment by PiotrMachowski** [comment]
https://github.com/PiotrMachowski/Home-Assistant-custom-components-Xiaomi-Cloud-Map-Extractor/issues/422
